hs_core/management/commands/compute_bag_checksum_for_published_resources.py

```diff
@@ -0,0 +1,25 @@
+
+from django.core.management.base import BaseCommand
+
+from hs_core.models import BaseResource
+from hs_core.hydroshare.utils import get_resource_by_shortkey
+from hs_core.tasks import create_bag_by_irods
+
+
+class Command(BaseCommand):
+    help = "Generate bag checksums for published resources as needed"
+
+    def add_arguments(self, parser):
+
+        # a list of resource id's: none does nothing.
+        parser.add_argument('resource_ids', nargs='*', type=str)
+
+    def handle(self, *args, **options):
+        if len(options['resource_ids']) > 0:  # an array of resource short_id to check.
+            for rid in options['resource_ids']:
+                resource = get_resource_by_shortkey(rid)
+                if resource.raccess.published:
```

Comment:
This just ignores a resource passed by the user if the resource isn't published.  It should include some feedback.

Reply:
Added a print statement for feedback.